FileShareClient/FileShareApiClient.cs

```diff
@@ -190,6 +191,28 @@ public async Task<IEnumerable<string>> GetUserAttributesAsync()
             }
         }
 
+        public async Task<IResult<BatchAttributesSearchResponse>> BatchAttributeSearchResponse(string searchQuery, CancellationToken cancellationToken)
```

Comment:
Please change the name back to `BatchAttributeSearch`
```suggestion
        public async Task<IResult<BatchAttributesSearchResponse>> BatchAttributeSearch(string searchQuery, CancellationToken cancellationToken)
```

Reply:
Updated as suggested. Thanks